xml/System.Numerics/Vector.xml

```diff
@@ -1870,7 +1870,7 @@ Note that this method returns a <xref:System.Single> instead of an integral type
         <param name="right">The second source vector.</param>
         <summary>Creates a new single-precision vector with elements selected between two specified single-precision source vectors based on an integral mask vector.</summary>
         <returns>The new vector with elements selected based on the mask.</returns>
-        <remarks>To be added.</remarks>
+        <remarks>This is equivalent to <paramref name="condition" /> <c>?</c> <paramref name="left" /> <c>:</c> <paramref name="right" /> on a per-bit basis.</remarks>
```

Comment:
We should be more specific about what "this" is. For example, "The returned vector is equivalent...".